jaeger-core/src/test/java/io/jaegertracing/reporters/RemoteReporterTest.java

```diff
@@ -242,14 +250,21 @@ public int flush() throws SenderException {
       }
     };
 
-    reporter = new RemoteReporter(sender, flushInterval, maxQueueSize, metrics);
-    tracer =
-          new Tracer.Builder("test-remote-reporter", reporter, new ConstSampler(true))
-                .withMetrics(metrics)
-                .build();
+    reporter = new RemoteReporter.Builder()
+            .withSender(sender)
+            .withFlushInterval(flushInterval)
+            .withMaxQueueSize(maxQueueSize)
+            .withMetrics(metrics)
+            .build();
+
+    tracer = new Tracer.Builder("test-remote-reporter")
+            .withReporter(reporter)
+            .withSampler(new ConstSampler(true))
+            .withMetrics(metrics)
+            .build();
 
     tracer.buildSpan("mySpan").start().finish();
-    latch.await(1, TimeUnit.SECONDS);
+    latch.await(2, TimeUnit.SECONDS);
```

Comment:
@kevinearls this does not solve the issue. You can call wait on the latch with a reasonable time.